lms/djangoapps/commerce/views.py

```diff
@@ -111,7 +111,8 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
 
         try:
             timeout = getattr(settings, 'ECOMMERCE_API_TIMEOUT', 5)
-            response = requests.post(url, data={'sku': course_modes[0].sku}, headers=headers, timeout=timeout)
+            response = requests.post(url, data=json.dumps({'sku': course_modes[0].sku}), headers=headers,
```

Comment:
Lack of understanding on my part. `CourseMode`'s unique key is on course_id, mode_slug, and currency. Are we going to have a SKU for every currency? If so, how do I determine the user's currency at this point? If not, this code is fine as the filter *should* only have one `CourseMode`. See https://github.com/edx/edx-platform/pull/7333/files#diff-3e84d5041cfb763b355bac2d2d063ae9R94.